### ISSUE TITLE
Focus & keyboard events

### ISSUE THREAD

**Issue by broweratcognitecdotcom** [johnwalley/d3-simple-slider#37]
It would be very good if I could figure out how to give the slider focus and have it respond to keyboard events such as left & right or up & down arrows.

**Comment by johnwalley** [comment]
I've been working on adding keyboard support and generally improving accessibility recently (see #18 for some previous discussion). I've just released this as version 1.5.0 which should be live.

In particular this lets you tab select the handle of the slider and use the keyboard to move it. Left, Right, Up, Down, PgUp, PgDown, Home and End are supported.

If you get the chance to try it out I'd be interested to get your feedback.
